patches/cli/0002-Make-BuildFromSource-disable-ASP.NET-dev-certs.patch

```diff
@@ -0,0 +1,100 @@
+From cd96ec40e18bf5cdce6cb3aad8b699051da7bf34 Mon Sep 17 00:00:00 2001
```

Comment:
I think we should also [patch this](https://github.com/dotnet/cli/blob/master/src/Microsoft.DotNet.Configurer/DotnetFirstTimeUseConfigurer.cs#L91-L99):

```
        private bool ShouldGenerateAspNetCertificate()
        {
#if DOTNET_BUILD_FROM_SOURCE
            return false;
#else
            var generateAspNetCertificate =
                _environmentProvider.GetEnvironmentVariableAsBool("DOTNET_GENERATE_ASPNET_CERTIFICATE", true);

            return ShouldRunFirstRunExperience() &&
                generateAspNetCertificate &&
                !_aspNetCertificateSentinel.Exists();
#endif
        }
```

to prevent the first run experience from telling users a certificate was created when one wasn't.

Reply:
Filed https://github.com/dotnet/source-build/issues/434 to get this in.